src/index.js

```diff
@@ -1,6 +1,7 @@
 var multiaddr = require('multiaddr')
 var getConfig = require('./config')
 var getRequestAPI = require('./request-api')
+var request = require('request')
```

Comment:
I'm not sure if request works in the browser. But the tests are testing that this actually works no?